services/morph_chain/protocol.privnet.yml

```diff
@@ -30,7 +30,7 @@ ApplicationConfiguration:
   RPC:
     Enabled: true
     EnableCORSWorkaround: false
-    MaxGasInvoke: 15
+    MaxGasInvoke: 50
```

Comment:
Mainnet has 100. Think, we can safely mirror this.

Reply:
Good idea. Changed.